style.css

```diff
@@ -0,0 +1,1427 @@
+  font-size: 0.8rem;
+  padding-left: 0.5rem;
+  text-decoration: none;
+}
+
+.top-second {
+  justify-content: space-between;
+  padding: 0.8rem;
+  background-color: rgb(253, 253, 253);
+  display: flex;
+}
+
+.top-second-left a,
+.top-second-right a {
+  color: rgb(131, 131, 131);
+  font-size: 0.8rem;
+  font-weight: 200;
+  padding-right: 0.4rem;
+  text-decoration: none;
+}
+
+.top-second-right a {
+  padding-left: 5px;
+  border-left: 1px solid rgb(131, 131, 131);
+  padding-right: 20px;
+}
+
+.top-third {
+  padding: 0.4rem;
+  font-size: 0.8rem;
+  background-color: #ffe600;
+}
+
+.top-third > a {
+  text-decoration: none;
+  padding-left: 0.4rem;
+}
+
+.section-one {
+  width: 100%;
+  height: 70vh;
+  grid-template-columns: 2fr 1fr;
+  grid-gap: 5px;
+  margin-bottom: 8vh;
+  margin-right: 3px;
+  margin-left: 3px;
+  margin-top: 6rem;
+  display: grid;
+}
+
+.first {
+  grid-area: 1 / 1 / 3 / 2;
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 40%, rgba(255, 68, 51, 0.8) 100%),
+    url("./images/Opal-796x417.jpg");
+  background-size: cover;
+  color: #fff;
+  position: relative;
+}
+
+.tnw-logo {
+  margin: 20px 0 0 20px;
+  width: 50%;
+}
+
+.second {
+  grid-area: 1 / 2 / 2 / 3;
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 50%),
+    url("./images/Apps.jpeg");
+  background-size: cover;
+  color: #fff;
+  padding: 1rem;
+  position: relative;
+}
+
+.third {
+  grid-area: 2 / 2 / 3 / 3;
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 50%),
+    url("./images/MacApp.jpg");
+  background-size: cover;
+  color: #fff;
+  position: relative;
+}
+
+.first-text,
+.second-text,
+.third-text {
+  bottom: 0;
+  left: 0.8rem;
+  position: absolute;
+}
+
+.first-text h1 {
+  font-weight: 900;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 200%;
+}
+
+.first-text p {
+  font-size: 80%;
+}
+
+.second-text h1,
+.third-text h1 {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 900;
+}
+
+.second-text p,
+.third-text p {
+  font-size: 60%;
+}
+
+.first-text a,
+.second-text a,
+.third a {
+  text-decoration: none;
+  color: white;
+}
+
+.ln-head-text {
+  font-weight: 900;
+  margin-left: 6rem;
+  color: #f43;
+}
+
+.latest-news {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  grid-gap: 2px 10px;
+  margin-bottom: 8vh;
+  width: 87%;
+}
+
+.latest-news > div {
+  margin-bottom: 10px;
+}
+
+.latest-news > div > img {
+  width: 270px;
+}
+
+.news-img-1 {
+  grid-area: 1 / 1 / 2 / 2;
+}
+
+.news-img-3 {
+  grid-area: 1 / 3 / 2 / 4;
+}
+
+.news-img-4 {
+  grid-area: 1 / 4 / 2 / 5;
+}
+
+.news-img-5 {
+  grid-area: 2 / 1 / 3 / 2;
+}
+
+.news-img-6 {
+  grid-area: 2 / 2 / 3 / 3;
+}
+
+.news-img-7 {
+  grid-area: 2 / 3 / 3 / 4;
+}
+
+.news-img-8 {
+  grid-area: 2 / 4 / 3 / 5;
+}
+
+.news-img-2 {
+  grid-area: 1 / 2 / 2 / 3;
+}
+
```

Comment:
## Thumbs up 👍 
✅ Great job showcasing the greatness that comes with arranging items on the UI using the `grid` box rule. I love how you pulled this off in a neat and minimalistic way. Awesome stuff.